provider/resource_keycloak_identity_provider_token_exchange_scope_permission.go

```diff
@@ -0,0 +1,292 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"log"
+	"strconv"
+	"strings"
+)
+
+func resourceKeycloakIdentityProviderTokenExchangeScopePermission() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceKeycloakIdentityProviderTokenExchangeScopePermissionCreate,
+		Read:   resourceKeycloakIdentityProviderTokenExchangeScopePermissionRead,
+		Delete: resourceKeycloakIdentityProviderTokenExchangeScopePermissionDelete,
+		Update: resourceKeycloakIdentityProviderTokenExchangeScopePermissionUpdate,
+		// This resource can be imported using {{realmId}}/{{providerAlias}}. The provider alias is displayed in the URL when editing it from the GUI
+		Importer: &schema.ResourceImporter{
+			State: resourceKeycloakIdentityProviderTokenExchangeScopePermissionImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"realm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"provider_alias": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"policy_type": {
```

Comment:
you could add some validation on this attribute if only `client` is supported, similar to https://github.com/mrparkers/terraform-provider-keycloak/blob/5d0c60bac0759dbdf26e1aee6693346b4b598fc9/provider/resource_keycloak_openid_client.go#L54

Reply:
Yes I will take a look